rubble/src/att/mod.rs

```diff
@@ -595,13 +951,26 @@ impl<A: AttributeProvider> AttributeServer<A> {
                     Err(RspError(e)) => Err(e),
                 }
             }
-            AttMsg::ExchangeMtuReq { mtu: _mtu } => {
-                responder
-                    .respond(OutgoingPdu(AttMsg::ExchangeMtuRsp {
-                        mtu: u16::from(Self::RSP_PDU_SIZE),
-                    }))
+
+            AttMsg::ReadReq { handle } => {
+                self.attrs
+                    .for_each_attr(&mut |att: &mut Attribute<'_>| {
+                        // Handles are unique so this can only occur once
+                        if att.handle == handle {
+                            responder
+                                .respond(OutgoingPdu(AttMsg::ReadRsp { value: att.value }))
+                                .unwrap();
+                            return Ok(());
```

Comment:
This is redundant with the `Ok(())` below. Did you mean to abort the iteration here? Then you need to return an `Err`, I believe.